license/unipdf_license_key_usage_log.go

```diff
@@ -0,0 +1,116 @@
+/*
+ * This example shows how to get license key usage logs
+ *
+ * Run as: go run unipdf_license_usage_log.go input_dir output_dir
+ */
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/unidoc/unipdf/v3/common"
+	"github.com/unidoc/unipdf/v3/common/license"
+	"github.com/unidoc/unipdf/v3/extractor"
+	"github.com/unidoc/unipdf/v3/model"
+)
+
+func init() {
+	// Make sure to load your metered License API key prior to using the library.
+	// If you need a key, you can sign up and create a free one at https://cloud.unidoc.io
+	err := license.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		panic(err)
+	}
+	license.SetMeteredKeyUsageLogVerboseMode(true)
+	common.SetLogger(common.NewConsoleLogger(common.LogLevelInfo))
+	license.SetMeteredKeyPersistentCache(false)
```

Comment:
Probably can remove this? as this only needed when to report the usage to unicloud server immediately

Reply:
Do you mean this one 
`license.SetMeteredKeyPersistentCache(false)` ?

Reply:
Yes